tests/conftest.py

```diff
@@ -249,6 +249,9 @@ def pytest_sessionstart(session: pytest.Session) -> None:
     ui_tests.read_fixtures()
     if session.config.getoption("ui") and _is_main_runner(session):
         testreport.clear_dir()
+        # Preparing a new file for UI diff
+        with open(ui_tests.FIXTURES_DIFF, "w") as _:
+            pass
 
```

Comment:
```suggestion
ui_tests.FIXTURES_DIFF.touch()
```

Reply:
That is unfortunately not possible here, as we want the file to be always empty at the beginning - not containing the old diff

Reply:
in that case `ui_tests.FIXTURES_DIFF.write_bytes(b"")`

Reply:
Done like this in force-push